google/cloud/bigtable/benchmarks/apply_read_latency_benchmark.cc

```diff
@@ -175,6 +175,8 @@ int main(int argc, char* argv[]) try {
 }
 
 namespace {
+namespace bigtable = google::cloud::bigtable;
```

Comment:
If you put this at the top of the file, then you don't have to modify any code (`bigtable::benchmarks` still works), and we can drop the `using google::cloud::bigtable::benchmarks` at the top of the file as well.

Thoughts?

Reply:
Done, in this file and other benchmarks.  I looked everywhere else and did not find cases where the changes could be minimized with similar techniques, I might have missed some :disappointed: